_core/smart/overview.md

```diff
@@ -1,135 +1,55 @@
-
-You can add comments to your Clarity contracts using  `;;` (double semi-colons). Both standalone and inline comments are supported.
-
-```cl
-;; Transfers tokens to a specified principal
-(define-public (transfer (recipient principal) (amount int))
-  (transfer! tx-sender recipient amount)) ;; returns: boolean
-```
+## Language design
 
-You use the `clarity-cli` command to check and launch a Clarity (`.clar`) program.
+Clarity is a [list processing (LISP) language](https://en.wikipedia.org/wiki/Lisp_(programming_language)) and differs from most other smart contract languages in two essential ways:
 
-## hello-world example
-
-The easiest program to run in any language is a hello world program. In Clarity, you can write this `hello-world.clar` program.
-
-```cl
-(begin 
-   (print "hello world"))
-```
-
-This program defines a single `hello-world` expression that is excuted when the contract launches. The `begin` is a native Clarity function that evaluates the expressions input to it and returns the value of the last expression.  Here there is a single `print` expression. Both the `begin` and the `print` are enclosed in `()` parentheses.
-
-For the pre-release, the Blockstack test environment includes the `clarity-cli` command for interacting with the contract and SQLite to support the data space. You create a SQLLite database to hold data related to Clarity contracts. This database simulates the blockchain by recording the contract activity. 
-
-You can't run even an a hello-world program without first initializing a Clarity contract's data space within the database. You can use the `clarity-cli initialize` command to set up the database.
-
-```clarity-cli initialize /data/db```
-
-This command initializes the `db` database which resides in the `/data` directory of the container. You can name the database anything you like, the name `db` is not required. You can use SQLite to query this database:
-
-```sql
-sqlite> .open db
-sqlite> .tables
-contracts            maps_table           type_analysis_table
-data_table           simmed_block_table 
-sqlite> 
-```
-
-After you initialize the contract's data space, you can `check` a Clarity program for problems.
-
-```clarity-cli check ./hello.clar /data/db```
-
-As the name implies, the `check` ensures the contract definition passes a type check; passing programs will returns an exit code of `0` (zero). Once a contract passes a check, you `launch` it.
-
-```bash
-root@4224dd95b5f5:/data# clarity-cli launch hello ./hello.clar /data/db
-Buffer(BuffData { data: [104, 101, 108, 108, 111, 32, 119, 111, 114, 108, 100] })
-Contract initialized!
-```
-
-Because Clarity does not support simple strings, it stores the `hello world` string in a buffer. Printing out that string displays the ASCII representation for each character. You can see the record of this contract's launch in the corresponding database:
+* The language is not intended to be compiled
```

Comment:
I think affirmative wording would sound better here, since we don't want to be framing our key features to sound like limitations.  We've used `The language is interpreted` and `The language is decidable` as alternatives in most other places (i.e. in talks, blog posts, tweets, etc.).

Reply:
Yeah -- I'd probably phrase this as

* ~The language is not intended to be compiled.~ The language is designed to be broadcast on the blockchain as is.
* ~The language is not Turing complete~ The language is decidable.

Reply:
Cool, will update